christmas/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS Christmas</title>
+    <link rel="icon" href="../favicon.ico">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="script.js"></script>  
+</head>
+<body onresize="resize()">
+    <noscript>
+      <meta http-equiv="refresh" content="0;url=noscript.html">
+    </noscript>
+    <div class="envelope new">
+        <div class="front">
+            <div class="mailme">
+            <p>A special message from CSS</p>
+            <p>(click!)</p>
+            </div>
+        </div>
+        <div class="back">
+            <div class="letter">
+              <img src="chrimsas.jpg" alt="" style="height: 75%; border-radius: 5px; position: absolute; left: 5px; top: 45px;">
+              <div style="position: absolute; right: 10px; top: 50px; width: 46%; text-align: center;">
```

Comment:
This is using inline styling. Why not extract it to the CSS import?
Also the left of the image and right of the text is different at 5 and 10px.

Reply:
Fixed.